server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -210,6 +219,10 @@ private String convertToString(ArrayList<String> comments) {
 
     for (String comment : comments) {
       comment = comment.replace("\"", "");
+      
+      if (comment.length() - 1 <= 0){
```

Comment:
`comment.length() <= 1` is more readable???

Reply:
You should also leave a comment why you're skipping 1 and 0 letter long comments

Reply:
Please add a comment. Something like "One letter strings can't be properly concatenated with the other comments because [reasons]". Right now I'm not sure why you're skipping one letter comments.